library/libraries.js

```diff
@@ -1078,5 +1078,15 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             }
             return false;
         }
+    },
+    'ScrollMagic': {
+        icon: 'scrollmagic',
+        url: 'http://scrollmagic.io/',
+        test: function(win) {
+            if (win.ScrollMagic && win.ScrollMagic.version) {
```

Comment:
Please use one of the non-`version` properties from the `ScrollMagic` object when checking for existence of ScrollMagic (such as `Controller`: http://scrollmagic.io/docs/ScrollMagic.Controller.html)